docs/documentation/aurora-config.md

```diff
@@ -553,46 +553,75 @@ or in the _env_ file if they should be used for all applications in that environ
+**Note**: Log files are expected to reside under `/u01/logs`
 
 #### Simple configuration
+
 A minimal configuration specifying the log index can be added to the configuration.
+
 ```yaml
-logging:              # configuration block
-  index: myapp-index  # splunk index where logs will be indexed 
+logging: # configuration block
+  index: myapp-index # splunk index where logs will be indexed
 ```
 
 #### Separate indexes
+
 It is possible to define separate splunk indexes for different types of logs.
 Logs that are not specified explicitly will be indexed to the index specified in `logging.index`.
 
 Available defined log types:
 
-| Name          | Type              | Log file pattern  |
-|------------------|----------------|-------------------|
-| access        | access_combined   | *.access          |
-| application   | log4j             | *.log             |
-| audit_json    | _json             | *.audit.json      |
-| audit_text    | log4j             | *.audit.text      |
-| gc            | gc_log            | *.gc              |
-| sensitive     | log4j             | *.sensitive       |
-| slow          | log4j             | *.slow            |
-| stacktrace    | log4j             | *.stacktrace      |
+| Name        | Type            | Log file pattern |
+| ----------- | --------------- | ---------------- |
+| access      | access_combined | \*.access        |
+| application | log4j           | \*.log           |
+| audit_json  | \_json          | \*.audit.json    |
+| audit_text  | log4j           | \*.audit.text    |
+| gc          | gc_log          | \*.gc            |
+| sensitive   | log4j           | \*.sensitive     |
+| slow        | log4j           | \*.slow          |
+| stacktrace  | log4j           | \*.stacktrace    |
 
 To configure a specific, of the available log types, then `logging.loggers.<logname>` must be configured (replace `<logname>` with one of the log type names).
 
 ```yaml
 logging:
-  index: myapp-index                    # splunk index where log will be indexed
+  index: myapp-index # splunk index where log will be indexed
   loggers:
-    stacktrace: myapp-stacktrace-index  # splunk index where stacktraces will be indexed
-    audit_text: aud-myapp-inxed         # splunk index where audit_text logs will be indexed
+    stacktrace: myapp-stacktrace-index # splunk index where stacktraces will be indexed
+    audit_text: aud-myapp-inxed # splunk index where audit_text logs will be indexed
+```
+
+#### Separate indexes with custom file pattern
+
+We have support for specifying custom file patterns if you do not follow the default patterns. We strongly advice you to use the default loggers config if possible.
+We do not allow custom sourcetypes, we currently support [_json, access_combined, gc_log, log4j].
+If you need support for another sourcetype you need to contact us.
+When you use custom logging, you practically takes full ownership of capturing logs with correct file pattern.
```

Comment:
`you practically takes full ownership` kan endres til `you take full ownership and responsibility`

Reply:
fikset